src/zenml/model/utils.py

```diff
@@ -33,12 +33,12 @@
 
 
 def link_step_artifacts_to_model(
```

Comment:
Should we rename this to `link_step_artifact_versions_to_model` as well?

Reply:
We could do, no strong opinion against it